ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -382,12 +378,7 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
         .filter(owner => owner.action !== ActionTypes.ADDED)
         .length === 1
     },
-    hasAddedExecutorsInGroup: (groupId): boolean => {
-      return getMhrTransferHomeOwnerGroups.value
-        .find(group => group.groupId === groupId).owners
-        .some(owner => owner.partyType === HomeOwnerPartyTypes.EXECUTOR &&
-          owner.action === ActionTypes.ADDED)
-    },
+    hasAddedExecutorsInGroup: (groupId): boolean => hasAddedpartyTypeToGroup(groupId, HomeOwnerPartyTypes.EXECUTOR),
```

Comment:
small thing but `hasAddedpartyTypeToGroup` should have word Party capitalized

Reply:
Thanks will update that!